pykickstart/commands/bootloader.py

```diff
@@ -211,7 +205,8 @@ class F12_Bootloader(F8_Bootloader):
 
     def _getParser(self):
         op = F8_Bootloader._getParser(self)
-        op.add_argument("--lba32", dest="forceLBA", deprecated=True, action="store_true")
+        op.add_argument("--lba32", dest="forceLBA", action="store_true",
+                        deprecated=True, version="Fedora12")
```

Comment:
I really don't like introducing these version= arguments (and the same sort of thing with remove_argument).  We already know which version added/removed/deprecated the argument - the version of the class the call was made from.  It seems likely there's some way we can automatically get this information.